src/test/java/com/jcabi/github/RtOrganizationsTest.java

```diff
@@ -89,61 +73,37 @@ public void retrievesOrganizations() throws Exception {
     }
 
     /**
-     * RtOrganizations can iterate organizations for an unauthenticated user.
+     * RtOrganizations should be able to iterate
+     * the logged-in user's organizations.
      *
      * @throws Exception If a problem occurs
+     * @checkstyle MagicNumberCheck (25 lines)
      */
     @Test
-    public void canIterateOrganizationsForUnauthUser() throws Exception {
+    public void retrievesOrganizations() throws Exception {
+        final Github github = new MkGithub();
         final MkContainer container = new MkGrizzlyContainer().next(
             new MkAnswer.Simple(
                 HttpURLConnection.HTTP_OK,
                 Json.createArrayBuilder()
-                    .add(org(Tv.THREE, "org11"))
-                    .add(org(Tv.FOUR, "org12"))
-                    .add(org(Tv.FIVE, "org13"))
+                    .add(org(1, "org1"))
+                    .add(org(2, "org2"))
+                    .add(org(3, "org3"))
                     .build().toString()
             )
         ).start();
         try {
             final Organizations orgs = new RtOrganizations(
-                new MkGithub(),
-                new ApacheRequest(container.home()),
-                Mockito.mock(User.class)
+                github,
+                new ApacheRequest(container.home())
             );
-            final String username = "octopus";
             MatcherAssert.assertThat(
-                orgs.iterate(username),
-                Matchers.<Organization>iterableWithSize(Tv.THREE)
+                orgs.iterate(),
+                Matchers.<Organization>iterableWithSize(3)
```

Comment:
@krzyk "Leave" isn't quite accurate (this diff is misleading), but changed this to `Tv.THREE` anyway.